src/table/components/TableHeadWrapper.jsx

```diff
@@ -18,13 +18,19 @@ function TableHeadWrapper({
   translator,
   selectionsAPI,
   keyboard,
-  children,
+  updateCellHeight,
 }) {
   const { columns, paginationNeeded } = tableData;
   const isFocusInHead = useContextSelector(TableContext, (value) => value.focusedCellCoord[0] === 0);
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme]);
 
+  useEffect(() => {
+    const height = rootElement.getElementsByClassName('sn-table-head-cell')[0].offsetHeight - 0.5;
```

Comment:
I know subtracting 0.5px might not make sense, it's just to temporarily fix the tiny gap between head cells and total cells. I should probably find a better solution since it might be different regarding the screen size and so on.

Reply:
That could be fine for now I guess. use the setter from the context here instead as Li did: https://github.com/qlik-oss/sn-table/pull/591/files